projects/lib/src/board/westernboard.cpp

```diff
@@ -165,6 +173,11 @@ QString WesternBoard::sanMoveString(const Move& move)
 
 	if (piece.type() == Pawn)
 	{
+		if ( m_pawnAmbiguous )
```

Comment:
OK, I suspected this. Will be corrected. Of course a coding style with spaces is more readable at times.